unconference/Project_taxonomy_recommendations.md

```diff
@@ -541,6 +533,12 @@ GenOSS: **Gen**eral Guidelines for an Inclusive **OSS** Project
 <!-- Footnotes themselves at the bottom. -->
 ## Notes
 
+[^5]: [https://github.com/badging/project-diversity-and-inclusion](https://github.com/badging/project-diversity-and-inclusion)
+[^6]: [https://doctorow.medium.com/the-title-card-from-sumana-harihareswaras-github-talk-what-would-open-source-look-like-if-it-were-35b0360f8b89](https://doctorow.medium.com/the-title-card-from-sumana-harihareswaras-github-talk-what-would-open-source-look-like-if-it-were-35b0360f8b89)
+[^7]: [https://www.jmlr.org/mloss/mloss-info.html](https://www.jmlr.org/mloss/mloss-info.html)
+
+## References
```

Comment:
These seem to have been swapped. The moved links are references and have no superindex within the text linking here whereas the ones already here are footnotes (at least formatting wise). We could merge it and see how things end up being formatted, I don't think I'll be able to check building it locally anytime soon